.github/workflows/close-stale-prs.yaml

```
@@ -0,0 +1,29 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '0 6 * * MON'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  close-stale-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@1160a2240286f5da8ec72b1c0816ce2481aabf84 # v8.0.0
+        with:
+          stale-pr-message: |
+            This PR is being labeled as "stale" because it has not been updated in 30 or more days.
+            
+            If this PR is still relevant, please remove the "stale" label. Otherwise this PR may be automatically closed after 14 days. If this PR is blocked, please add it to the "Blocked" milestone.
+          close-pr-message: |
+            This PR has been automatically closed because it has been "stale" for 14 or more days. Please feel free to re-open this PR if it is still relevant.
+          stale-pr-label: 'stale'
+          only-labels: 'waiting-response'
+          exempt-pr-labels: 'breaking-change,upstream/microsoft,upstream/pandora,upstream/terraform'
+          days-before-pr-stale: 30
+          days-before-pr-close: 14
```

```suggestion
            This PR has been automatically closed because it has been "stale" for 30 or more days. Please feel free to re-open this PR if it is still relevant.
          stale-pr-label: 'stale'
          only-labels: 'waiting-response'
          exempt-pr-labels: 'breaking-change,upstream/microsoft,upstream/pandora,upstream/terraform'
          days-before-pr-stale: 30
          days-before-pr-close: 30
```

I think we should make this 30  + 30